deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml

```diff
@@ -2,13 +2,13 @@ version: '2.1'
 
 services:
   zookeeper:
-    image: wurstmeister/zookeeper
+    image: registry-mirror.pingcap.net/wurstmeister/zookeeper
```

Comment:
This is not consistent with the address used in CI, plus using it may not be friendly to non-Chinese network environments?

Reply:
It looks like the external network cannot access this address.